website/docs/advanced/feature-toggle-variants.md

```diff
@@ -2,24 +2,100 @@
+## What are variants?
 
-![toggle_variants](/img/variants.png 'Feature Toggle Variants')
+Whenever you create a feature toggle, you can assign it any number of _variants_. This is commonly done in cases where you want to serve your users different versions of a feature to see which performs better.
+
+A variant has four components that define it:
+- a **name**:
+
+    This must be unique among the toggle's variants. When working with a toggle with variants in a client, you will typically use variant name to find out which variant it is.
+
+- a **weight**:
+
+    The weight is the likelihood of any one user getting this specific variant. See the [weights section](#variant-weight) for more info.
+
+- an optional **payload**:
+
+    A variant can also have an associated payload. Use this to deliver more data or context. See the [payload section](#variant-payload) for a more details.
+
+
+- an optional **override**
+
+    Overrides let you specify that certain users (as identified either by their user ID or by another [custom stickiness](stickiness) value) will always get this variant, regardless of the variant's weighting.
+
+![A form for adding new variants. It has fields for name, weight, payload, and overrides.](/img/variant-creation-form.png 'Creating a new toggle variant')
+
+### Variant weight
+
+A variant's weight determines how likely it is that a user will receive that variant. It is a numeric value between 0 and 100 (inclusive) with one decimal's worth of precision.
+
+When you have multiple variants, the sum of all their weights must add up to exactly 100. Depending on the [weight type](#weight-types), Unleash may automatically determine the weight of the new variant and balance it out with the other variants.
+
+#### Weight types and calculation
+
+There are two kinds of variant weight types: _variable_ and _fixed_. Unleash requires you to always have _at least_ one variable weight variant.
+
+The default weight type is _variable_. Variable weight variants will adjust their weight based on the number of other variable weight variants and whatever weight is not used up by fixed weight variants.
+
+_Fixed_ weight variants have a set weight which will not change. All fixed weight variants' weights can not add up to more than 100.
+
+To calculate what the weight of a variable weight variant is, Unleash first subtracts the sum of fixed weights from 100 and then distributes the remaining weight evenly among the variable weight variants.
 
-#### How does it work? {#how-does-it-work}
+For instance, if you have three variable weight variants and two fixed weight variants weighted at 25 and 15 respectively, Unleash will:
+1. Subtract the fixed weight from the total available: 100 - 40 = 60
+2. Divide the remainder by the number of variable weight variants: 60 / 3 = 20
+3. Assign each variable weight variant the same (up to rounding differences) weight: 20%
```

Comment:
So unleash handles rounding now all the way to 100.0 percent. If you have 3 variable variants the weights they'll get will be 33.4, 33.3 and 33.3. I'm not sure how to phrase this well or even if this minor of a detail needs to get added?

Reply:
Aye, I was a bit torn about this. In the end I ended up going against it because it would make the description a bit longer and more cumbersome, but I got an idea now 🧠 Thanks for bringing it to my attention again.